torch/csrc/jit/lite_interpreter/bytecode.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
lite_interpreter_compilation_unit.h. All the APIs should match compilation unit.